pkg/nodeconfig/nodeconfig.go

```diff
@@ -129,6 +137,44 @@ func discoverKubeAPIServerEndpoint() (string, error) {
 	return host.Status.APIServerInternalURL, nil
 }
 
+// getWICDCredentials returns the CA cert and access token associated with the WICD service account
+func getWICDCredentials() (*authentication, error) {
+	cfg, err := crclientcfg.GetConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get config to talk to kubernetes api server")
+	}
+	clientset, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating kubernetes clientset")
+	}
+
+	secrets, err := clientset.CoreV1().Secrets("openshift-windows-machine-config-operator").List(context.TODO(),
```

Comment:
Yep, added a commit so we can reuse this

Reply:
Took the approach to read namespace from environment var since we can't always assume operator is deployed in `openshift-windows-machine-config-operator` namespace